ios/FluentUI/ResizingHandleView/ResizingHandleView.swift

```diff
@@ -5,30 +5,22 @@
 
 import UIKit
 
-// MARK: ResizingHandle Colors
-
-private extension Colors {
-    struct ResizingHandle {
-        public static var mark: UIColor = iconSecondary
-    }
-}
-
 // MARK: - ResizingHandleView
 
 @objc(MSFResizingHandleView)
-open class ResizingHandleView: UIView {
+open class ResizingHandleView: UIView, TokenizedControlInternal {
```

Comment:
What are your thoughts on the tokenization of the handle on the controls that use it? (bottom sheet and drawer controller). They currently allow the customization of the background color directly.

Would they be overriding the tokens of the ResizingHandle instead of setting those directly?

Reply:
Doesn't looks like bottom sheet is trying to set the background color anywhere, but I've added a deprecation message to the drawer and exposed the override tokens of the resizing handle view for them to use.

Reply:
Added some comments about my concerns of exposing the override tokens for the resizing handle from the Drawer implementation.